packages/@aws-cdk/aws-amplify/lib/domain.ts

```diff
@@ -21,6 +21,20 @@ export interface DomainOptions {
    * @default - use `addSubDomain()` to add subdomains
    */
   readonly subDomains?: SubDomain[];
+
+  /**
+   * Automatically create subdomains for connected branches
+   *
+   * @default false
+   */
+  readonly domainEnableAutoSubDomain: boolean;
```

Comment:
I think naming for these properties should be `enableAutoSubdomain` and `autoSubdomainCreationPatterns`. This matches the properties on the L1 and the `domain` prefix feels unnecessary.

Reply:
Agree, I tried to use the same props from `Domain`:

* domainEnableAutoSubDomain
* domainAutoSubDomainCreationPatterns

But I agree that the ones from `CfnDomain` are more clear:
* enableAutoSubDomain
* autoSubDomainCreationPatterns

I'll rename the new ones from `DomainOptions`

Do you think it would be ok to rename the ones from `Domain`? it breaks compatibility but it seems that they were never set, just empty read.

Reply:
Forget about my question, CfnDomain is generated from the specification 

Renamed the ones of amplify-cdk